radarr/movie.go

```diff
@@ -118,53 +123,17 @@ type AddMovieOptions struct {
 	SearchForMovie bool `json:"searchForMovie"`
 }
 
-// AddMovieOutput is the data returned when adding a movier.
-type AddMovieOutput struct {
```

Comment:
The only difference between this and `Movie` was the `AddOptions` member. I went ahead and added that to `Movie`.